docs/content/querying/lookups.md

```diff
@@ -0,0 +1,191 @@
+# Lookups
+
+Lookups are a concept in Druid where dimension values are (optionally) replaced with a new value.
```

Comment:
i think a lot more information is required on how lookups work (do they poll, etc)